### DISCUSSION TITLE
seeking further clarification to the meaning of "cancelling" a notice when some changes have been anounced before

### DISCUSSION THREAD

**Discussion by gabrieledall** [OP-TED/eForms-SDK#118] [Q&A] [answered]
Dear all, 

thank you very much for changing the labelling of the code "cancel" to the expression "Notice cancelled" instead of "cancellation" (in the codelist named "change-corrig-justification" and published on the EU-Vucabularies site but has not been changed in the codelist beeing pulished with the latest SDK 1.1.1). This makes clear that the canellation of a notice does not intend to cancel the whole procedure with one single notice.

At the moment we are discussing very much the handling of a chain of notices being built up by a sequence of change notices. I think we understood it so far - but then a new question came up: 

Someone pulished a notice and changed some of it's content by publishing change notices in the meantime - and then realizes that a canellcation is required: will the latest canellation-change-notice cancel the whole list of notices (from the first one to the last published change-notice) or will only the last "change"-Notice is expected to be cancelled?

I would expect the complete notice-chain would be cancelled but some of my peers had a different understanding and would only the last (changed) notice expect beeing cancelled.

Which expectation would be the correct one from your point of view?

Thank you very much for your answer.
Gabriele

**Comment by YvesJo** [comment, marked as answer]
Hi Gabriele,
You may find below some information

1. Once a notice has been changed, it can not be cancelled any more; also a change on a notice can not itself be cancelled.
2. If a notice is cancelled (hence a notice that did not have any applied change), then the process should resume at the stage just prior to its publication. Everything will be as if the notice didn't exist, except some general constraints that will remain (e.g. the notice identifiers of the notice and the cancel change notice can't be used anymore, etc).

KR,
Yves

**Reply by gabrieledall** [reply]
Hi Yves, 
thank you very much for your answer although I didn't expect it the way you put it. And therefore, I have two more questions regarding Point 1:
1. Changing a notice implies that the procurement process has to be completed reagularly afterwards - and therefore each changed notice will lead to the publication of some results (CAN) related to the changed notice. Is this correct?
2. Is using the Code "cancel-intend" intended to behave equally - is it also impossible to use this code on an notices that has already been changed before?


Kind regards
Gabriele

**Reply by YvesJo** [reply]
Hi,
1. Yes, the process should go on "regularly", and Result notices should exist wherever expected. When the Change is already on a Result notice, then there may be, in some cases, no need for any further Result notice.
2. "Cancel-intend" is there to indicate the intention to cancel the competition for a Lot or even the whole procedure. It is possible to use it on a notice previously changed as long as the change does not refer to a "cancel" (which would deem that notice as void).
